src/Gemstone.IO/Collections/FileBackedLookupTable.cs

```diff
@@ -1727,74 +1723,139 @@ static FileBackedLookupTable()
 
     // Static Methods
 
-    private static Action<Stream, T>? GetWriteMethod<T>()
+    private static Action<Stream, object>? GetWriteMethod(Type type)
     {
-        Type type = typeof(T);
-        MethodInfo? method = type.GetMethod("WriteTo", Flags, null, s_types, null);
+        MethodInfo? method = type.GetMethod("WriteTo", s_instanceFlags, null, s_types, null);
 
         if (method is not null)
-        {
-            Action<T, Stream> action = (Action<T, Stream>)Delegate.CreateDelegate(typeof(Action<T, Stream>), method);
-            return (stream, obj) => action(obj, stream);
-        }
+            return (stream, obj) => method.Invoke(obj, [stream]);
```

Comment:
This seems like it would be significantly slower than invoking a delegate created by `Delegate.CreateDelegate()`.

Reply:
Indeed - this got factored out during some updates. Will replace.